flaml/tune/tune.py

```diff
@@ -134,6 +136,7 @@ def run(
     metric_constraints: Optional[List[Tuple[str, str, float]]] = None,
     max_failure: Optional[int] = 100,
     use_ray: Optional[bool] = False,
+    use_incumbent_result=False,

```

Comment:
add docstr.
```suggestion
    use_incumbent_result_in_evaluation: Optional[bool] = None,
```
Only when it's set to True or False, we pass it to search_alg.